docs/src/app/(docs)/getting-started/solid/page.mdx

```diff
@@ -90,45 +88,54 @@ export type OurFileRouter = typeof uploadRouter;
 </Note>
 
 ```ts {{ title: "src/routes/api/uploadthing.ts" }}
+import type { APIEvent } from "@solidjs/start/server";
+
 import { createRouteHandler } from "uploadthing/server";
 
 import { uploadRouter } from "~/server/uploadthing";
 
-const handlers = createRouteHandler({
+const handler = createRouteHandler({
   router: uploadRouter,
-
-  // Apply an (optional) custom config:
-  // config: { ... },
 });
-export { handlers as GET, handlers as POST };
+
+export const GET = (event: APIEvent) => handler(event.request);
+export const POST = (event: APIEvent) => handler(event.request);
 ```
 
 > See configuration options in
 > [server API reference](/api-reference/server#create-route-handler)
 
-## Creating The UploadThing Components
+## Create The UploadThing Components
 
-Generating components let's you pass your generic `FileRouter` once and then
-have typesafe components everywhere, instead of having to pass the generic
-everytime you mount a component, but you can also import the components
-individually from `@uploadthing/solid`.
+We provide components to make uploading easier. We highly recommend re-exporting
+them with the types assigned, but you CAN import the components individually
+from `@uploadthing/solid` instead.
 
 ```ts {{ title: "src/utils/uploadthing.ts" }}
 import {
+  generateSolidHelpers,
   generateUploadButton,
   generateUploadDropzone,
 } from "@uploadthing/solid";
 
-import type { OurFileRouter } from "~/server/uploadthing";
+import type { UploadRouter } from "~/server/uploadthing";
 
 // URL is only needed for server side rendering, or when your router
 // is deployed on a different path than `/api/uploadthing`
 const url = `http://localhost:${process.env.PORT ?? 3000}`;
 
-export const UploadButton = generateUploadButton<OurFileRouter>({ url });
-export const UploadDropzone = generateUploadDropzone<OurFileRouter>({ url });
+export const UploadButton = generateUploadButton<UploadRouter>({ url });
+export const UploadDropzone = generateUploadDropzone<UploadRouter>({ url });
+export const { createUploadThing } = generateSolidHelpers<UploadRouter>({
+  url,
+});
 ```
 
+<Note>
+  To learn more about the components, check out the [solid API
+  Reference](/api-reference/solid).
```

Comment:
this page doesn't contain anything other than a redirect to React docs. did you mean to continue here or is that intentinoal?

Reply:
No, as you can see from the PR description:

> Plus added link to @uploadthing/solid api reference. (Which right now points to @uploadthing/react. I think it would be also okay to say directly in getting started that @uploadthing/solid reference is not available, but you can check out @uploadthing/react)

I guess there are two options:
- Leave reference to solid's api and if people click it, they will need to navigate to `api-reference/react`. I guess this way whenever solid api is added, this will work with no change.
- Add something like "To learn more about the components, check you API reference. The docs for `@uploadthing/solid` are currently not available, however the `@uploadthing/react` package is almost identical so so we recommend following those docs for now.". This is quite wordy and also would have to be changed in future. 

Don't know which is better. But I'm happy to change it.